source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -688,5 +688,45 @@ than an element in the range being searched.
 
 The elements e of [start, end) must be partitioned with respect to the comparator used.
 
+.. code:: cpp
+
+    template<typename Policy, typename InputKeyIt, typename InputValueIt,
+        typename Comparator =
+            std::less<typename std::iterator_traits<InputIt>::value_type>>
+    void
+    sort_by_key(Policy&& policy, InputKeyIt keys_first, InputKeyIt keys_last,
+                InputValueIt values_first, Comparator comp =
+                    std::less<typename std::iterator_traits<InputKeyIt>::value_type>());
+
+``oneapi::dpl::sort_by_key`` sorts a sequence of keys within the range ``[keys_first, keys_last)``.
+It also permutes the corresponding sequence of values within the range
+``[values_first, values_first + distance(keys_first, keys_last))``.
+Each key is associated with a corresponding value,
+and this association is maintained throughout the sorting process.
+
+The order of equal keys is not guaranteed to be preserved.
+
+If no comparator is provided, the keys are sorted with respect to ``std::less{}``.
+
+.. code:: cpp
+
+    template<typename Policy, typename InputKeyIt, typename InputValueIt,
+      typename Comparator =
+          std::less<typename std::iterator_traits<InputIt>::value_type>>
+    void
+    stable_sort_by_key(Policy&& policy, InputKeyIt keys_first, InputKeyIt keys_last,
+                InputValueIt values_first, Comparator comp =
+                    std::less<typename std::iterator_traits<InputKeyIt>::value_type>());
+
+``oneapi::dpl::stable_sort_by_key`` sorts a sequence of keys within the range ``[keys_first, keys_last)``.
+It also permutes the corresponding sequence of values within the range
+``[values_first, values_first + distance(keys_first, keys_last))``.
+Each key is associated with a corresponding value,
+and this association is maintained throughout the sorting process.
+
+The order of equal keys is preserved.
```

Comment:
```suggestion
Sorting is stable. That means, keys that do not precede one another according to the given comparator
and their associated values maintain their relative order as in the original sequences.
```

Reply:
What do you think about this small enhancement?

> Sorting is stable. That means, if keys do not precede one another according to the given comparator, their relative order in the original sequence is preserved, along with their associated values.

It should be more readable due to having "blocks" separated by commas.

Reply:
I've applied the suggestion as is. The sentence is long, but conveys the meaning well.

Reply:
The wording modification you suggested seems fine to me as well.

Reply:
There were more modifications of this part, so this discussion is not relevant anymore.